components/Nav/MobileNavbar.js

```diff
@@ -0,0 +1,154 @@
+import React, { PureComponent } from 'react'
+import styled, { css } from 'styled-components'
+
+import rem from '../../utils/rem'
+import { navbarHeight } from '../../utils/sizes'
+import { paleGrey } from '../../utils/colors'
+import { mobile } from '../../utils/media'
+import Link from '../Link'
+import NavLinks from './NavLinks'
+import Social from './Social'
+// import Search from './Search'
+import Logo from './Logo'
+import NavSeparator from './NavSeparator'
+import NavButton from './NavButton'
+
+const Wrapper = styled.div`
+  display: none;
+
+  ${mobile(css`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: ${rem(navbarHeight)};
+  `)}
+`
+
+const SecondaryMenu = styled.div`
+  position: absolute;
+  top: ${rem(navbarHeight)};
+  left: 0;
+  right: 0;
+
+  ${p => p.open ? css`
+    height: ${rem(navbarHeight)};
+  ` : css`
+    height: 0;
+  `}
+
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 ${rem(20)};
+  transition: height 0.1s;
+
+  -webkit-user-select: none;
```

Comment:
`user-select` is prefixed only for webkit here. The prefix can be removed

Reply:
Good catch!